src/controllers/dynakube/dynakube_controller.go

```diff
@@ -145,7 +135,9 @@ func (controller *Controller) reconcile(ctx context.Context, dynaKube *dynatrace
 	err := controller.reconcileDynaKube(ctx, dynaKube)
```

Comment:
line 163 (sorry can't comment on it directly) should never return both a result and an error

Reply:
@chrismuellner you mean this one 

https://github.com/Dynatrace/dynatrace-operator/blob/1ad499de870525707d6b2f4e1acdae6a003f8e0a/src/controllers/dynakube/dynakube_controller.go#L173

Reply:
either a result or an error is returned

Reply:
sorry, I mean either non-empty result + nil error  or empty result  + error